### ISSUE TITLE
How to change audio language

### ISSUE THREAD

**Issue by vikrant-pune** [botallen/repository.botallen#229]
Wanted to know if there is way to change audio language of channel

**Comment by vivinnl** [comment]
Yes there is, just click ok, hover over the house over the video feed, a seek bar appears and in that you will see a gear button, click on that button and it takes you to a option with "Audio settings" and there you can change the language of the channel if the channels supports it.
its just like changing the audio in Hotstar or Netflix, just explore the UI you have many useful features.
